### ISSUE TITLE
Documentation for Sonoff RF bridge needs to be updated

### ISSUE THREAD

**Issue by oldgeezy** [tasmota/docs#670]
@
re PR: arendst/tasmota#

Following the instructions here:

https://github.com/tasmota/docs/blob/master/docs/devices/Sonoff-RF-Bridge-433.md

I was not able to flash the RF chip.  I got an invalid file signature error trying to upload the hex files given by Tasmota.

When I downloaded the Portisch firmware directly from here:

https://github.com/Portisch/RF-Bridge-EFM8BB1/releases/tag/0x04

It worked just fine.

I would update it myself, but I don't know how, or perhaps have permission.

Regards

**Comment by barbudor** [comment]
@oldgeezy You probably didn't performed the download correctly
Steps are :
- Follow the link to https://github.com/arendst/Tasmota/tree/master/tools/fw_SonoffRfBridge_efm8bb1
- Click the the most recent firmware, as of today `RF-Bridge-EFM8BB1-20181023.hex`
- Once on the page of this firmware, click on the `Raw` button on the top right of the file display
![image](https://user-images.githubusercontent.com/2996042/110996031-980ae900-837b-11eb-937c-edf06c19ab4e.png)
- Once the HEX file is loaded in your browser, use your browser menu to save to your hard disk

Check by opening the file you previously downloaded in a text editor if it is an HEX file or if you misdownloaded an HTML file

If you want to add more details to the doc, please feel free by using the pencil icon on the top right of the doc page.

**Comment by oldgeezy** [comment]
You are correct sir!  That's frustrating - can't remember exactly which technique I used to download it.  Thought it was wget.  But opening the HEX file in a text editor clearly shows that it's an HTML file.

Also wonder which way I did it differently from the Portisch repo that I was actually able to get the HEX file...?

I'd add a precautionary note there if 'Edit the file in your fork of this project' (which is what shows when I hover over that edit button is the correct way.

Thank you!
